nss-sapphire-elephants/BangazonAPI
# Hannah product setup

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change


The GET, POST, PUT, and DELETE methods have been checked manually by myself as well as checked via integration tests.

# Testing Instructions for Change Made

```git fetch --all```
```git checkout hannah-product-setup```

- Please check 

- That the namespace for all files in BangazonAPI folder (controllers, models, program.cs and startup.cs) is ```BangazonSprint```, not ```BangazonSprintStartup```.
- That integration testing for TestProduct.cs is successful.
- That the ```README.md``` contains the correct details for ```Product.cs```


# Checklist:
